op-node/flags/p2p_flags.go

```diff
@@ -292,6 +292,7 @@ var (
 	SyncReqRespFlag = &cli.BoolFlag{
 		Name:     "p2p.sync.req-resp",
 		Usage:    "Enables experimental P2P req-resp alternative sync method, on both server and client side.",
```

Comment:
Probably shouldn't be a default if considered experimental. :)

```suggestion
		Usage:    "Enables P2P req-resp alternative sync method, on both server and client side.",
```

Reply:
hah - good spot. :)